infra/prod/_modules/functions/data.tf

```diff
@@ -0,0 +1,22 @@
+data "azurerm_application_insights" "application_insights" {
```

Comment:
why this file is called 'data'? looks like networking + ai

Reply:
I followed the same structure of https://github.com/pagopa/io-fims/tree/main/infra/prod/_modules/app_services

Reply:
Ah, now I get it, it's data from data sources. That's right!